server/index.js

```diff
@@ -1,6 +1,17 @@
 const express = require('express');
+const { Review } = require('./db.js');
 
 const app = express();
 const port = 3002;
 
+app.get('/abodes/:abode_id/reviews', (req, res) => {
+  Review.findOne({ abode_id: parseInt(req.params.abode_id) }, (err, abodeInfo) => {
```

Comment:
This might be a result of a temporary work in progress but the end point `/abodes/:abode_id/reviews` imply that we are fetching _all_ reviews. `findOne` is only looking for one document so we should appropriately rename our route or use the `findmany` funciton.